weather-forecast/.env

```diff
@@ -1,2 +1,2 @@
-REACT_APP_API_URL="https://api.openweathermap.org/data/2.5/weather"
+REACT_APP_API_BASE_URL="https://api.openweathermap.org/data/2.5/"
```

Comment:
Looks like this file is still in the git. Make sure it is ignored by running `git rm --cached .env`